packages/bot-web-ui/src/components/quick-strategy/quick-strategy.scss

```diff
@@ -18,6 +18,22 @@
     }
 }
 
+.dc-input {
+    .dc-input__field {
```

Comment:
Please add a `field_className` and remove this, then we can remove unnecessary `.dc-input` class usage also.

Reply:
You mean to add `field_className` as a new props also?

Reply:
Yes.